base/shell/explorer/traywnd.cpp

```diff
@@ -58,6 +58,57 @@ void appbar_notify_all(HMONITOR hMon, UINT uMsg, HWND hwndExclude, LPARAM lParam
 
 static const WCHAR szTrayWndClass[] = L"Shell_TrayWnd";
 
+struct WINDOWPOSBACKUPDATA
+{
+    HWND hwnd;
+    UINT x;
+    UINT y;
+    UINT cx;
+    UINT cy;
+};
+CSimpleArray<WINDOWPOSBACKUPDATA>  g_WindowPosBackup;
+
+struct WINDOWPOSBACKUPDATA_INFO
+{
+    CSimpleArray<WINDOWPOSBACKUPDATA> *pWindowsPosBackup;
+};
+
+static BOOL CALLBACK BackupWindowsPosProc(HWND hwnd, LPARAM lParam)
+{
+    WINDOWPOSBACKUPDATA_INFO *info = (WINDOWPOSBACKUPDATA_INFO *)lParam;
+    WINDOWPOSBACKUPDATA wposdata;
+    RECT rcWindow;
+    if (::IsWindowVisible(hwnd) && !::IsIconic(hwnd))
+    {
+        wposdata.hwnd = hwnd;
+        GetWindowRect(hwnd, &rcWindow);
```

Comment:
Why are `IsWindowVisible` and `IsIconic` prefixed with `::` and `GetWindowRect` not?